### PR TITLE
Add task for clean code block. 

### PR DESCRIPTION
**Pull request by lerik14**
Also add output method in HouseUtils class in java classes block to avoid duplications

### CODE REVIEW

File: clean_code_block/pom.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <artifactId>EPAM_course</artifactId>
+        <groupId>org.example</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>clean_code_block</artifactId>
+
+
+<!--    <properties>-->
```
Comment (Orkyboy) [inline clean_code_block/pom.xml]: if it is not needed - delete it

File: clean_code_block/src/main/java/Airport.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
+import Planes.ExperimentalPlane;
+import models.MilitaryType;
+import Planes.MilitaryPlane;
+import Planes.PassengerPlane;
+import Planes.Plane;
+
+import java.util.*;
+
+// version: 1.1
+// made by Vitali Shulha
```
Comment (Orkyboy) [inline clean_code_block/src/main/java/Airport.java]: I think that is version 1.2 made by Lera

File: java_classes_block/main_task/src/main/java/HouseUtils.java [hunk 3]
```diff
@@ -41,4 +41,14 @@ public static int readIntFromConsole() {
         }
         return number;
     }
+
+    public static void outputList(List<House> houseList) {
+        if (!houseList.isEmpty()) {
+            for (House house : houseList) {
+                System.out.println(house.toString());
+            }
+        }else {
```
Comment (Orkyboy) [inline java_classes_block/main_task/src/main/java/HouseUtils.java]: fix indentation pls

File: java_classes_block/main_task/src/main/java/HouseUtils.java [hunk 4]
```diff
@@ -41,4 +41,14 @@ public static int readIntFromConsole() {
         }
         return number;
     }
+
+    public static void outputList(List<House> houseList) {
```
Comment (Orkyboy) [inline java_classes_block/main_task/src/main/java/HouseUtils.java]: I'd call it `printHouseListToConsole`

File: pom.xml [hunk 5]
```diff
@@ -2,15 +2,21 @@
...
-
-    <groupId>org.example</groupId>
     <artifactId>EPAM_course</artifactId>
+    <groupId>org.example</groupId>
     <version>1.0-SNAPSHOT</version>
 
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>pom</packaging>
+
+
```
Comment (Orkyboy) [inline pom.xml]: empty line

**Review by Orkyboy — APPROVED** [review]

File: clean_code_block/pom.xml (hunk 1 above)
Comment (lerik14) [reply clean_code_block/pom.xml]: deleted

File: clean_code_block/src/main/java/Airport.java (hunk 2 above)
Comment (lerik14) [reply clean_code_block/src/main/java/Airport.java]: i deleted this info

File: java_classes_block/main_task/src/main/java/HouseUtils.java (hunk 3 above)
Comment (lerik14) [reply java_classes_block/main_task/src/main/java/HouseUtils.java]: done

File: java_classes_block/main_task/src/main/java/HouseUtils.java (hunk 4 above)
Comment (lerik14) [reply java_classes_block/main_task/src/main/java/HouseUtils.java]: renamed

File: pom.xml (hunk 5 above)
Comment (lerik14) [reply pom.xml]: deleted

File: clean_code_block/src/main/java/Airport.java [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+        PassengerPlane planeWithMaxCapacity = passengerPlanes.get(0);
+        for (int i = 0; i < passengerPlanes.size(); i++) {
+            if (passengerPlanes.get(i).getPassengersCapacity() > planeWithMaxCapacity.getPassengersCapacity()) {
+                planeWithMaxCapacity = passengerPlanes.get(i);
+            }
+        }
+        return planeWithMaxCapacity;
+    }
+
+    public List<MilitaryPlane> getTransportMilitaryPlanes() {
```
Comment (Orkyboy) [inline clean_code_block/src/main/java/Airport.java]: I'd combine these 2 methods together to avoid duplication
like DRY
```
public List<MilitaryPlane> getTransportMilitaryPlanes(MillitaryType type)
```

File: clean_code_block/src/main/java/Airport.java [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+            }
+        });
+        return this;
+    }
+
+    public List<? extends Plane> getPlanes() {
+        return planes;
+    }
+
+    private void print(Collection<? extends Plane> collection) {
```
Comment (Orkyboy) [inline clean_code_block/src/main/java/Airport.java]: rename

File: clean_code_block/src/main/java/Airport.java [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+    }
+
+    @Override
+    public String toString() {
+        return "Airport{" +
+                "Planes=" + planes.toString() +
+                '}';
+    }
+
+    public Airport(List<? extends Plane> planes) {
```
Comment (Orkyboy) [inline clean_code_block/src/main/java/Airport.java]: please move constructor to be a first method in the class

File: clean_code_block/src/main/java/Airport.java [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+    public Airport sortByMaxDistance() {
+        Collections.sort(planes, new Comparator<Plane>() {
+            public int compare(Plane o1, Plane o2) {
+                return o1.getMaxFlightDistance() - o2.getMaxFlightDistance();
+            }
+        });
+        return this;
+    }
+
+    public Airport sortByMaxSpeed() {
```
Comment (Orkyboy) [inline clean_code_block/src/main/java/Airport.java]: sortPlanesByMaxSpeed

**Review by Orkyboy — APPROVED** [review]

File: clean_code_block/src/main/java/Airport.java (hunk 9 above)
Comment (lerik14) [reply clean_code_block/src/main/java/Airport.java]: renamed

File: clean_code_block/src/main/java/Airport.java (hunk 8 above)
Comment (lerik14) [reply clean_code_block/src/main/java/Airport.java]: removed

File: clean_code_block/src/main/java/Airport.java (hunk 7 above)
Comment (lerik14) [reply clean_code_block/src/main/java/Airport.java]: done - printPlanes

File: pom.xml [hunk 10]
```diff
@@ -2,15 +2,20 @@
...
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>clean_code_block</module>
+    </modules>
+
     <properties>
-        <maven.compiler.source>8</maven.compiler.source>
-        <maven.compiler.target>8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
```
Comment (Orkyboy) [inline pom.xml:17]: was this change intended?

**Review by Orkyboy — APPROVED** [review]

File: pom.xml (hunk 10 above)
Comment (lerik14) [reply pom.xml:17]: Now it is the right version because I use java 8 that is why I should use 1.8 compiler
https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html
